critiquebrainz/frontend/templates/review/entity/base.html

```diff
@@ -143,28 +143,37 @@ <h3>
         <h3> Comments ({{ comment_count }})</h3>
         <div id="comment-list">
           {% for comment in comments %}
-          <div id="{{ comment.id }}">
-            <hr/>
-            <p>
-              {{ comment_credit(comment, user_picture_size=24) }}
-              <span style="float:right;">
-                <small><em class="text-muted"> {{comment.created | date}} </em></small>
-              </span>
-            </p>
-            <p> {{ comment.text_html|safe }} </p>
-          </div>
+            <div id="{{ comment.id }}">
+              <hr/>
+              <p>
+                {{ comment_credit(comment, user_picture_size=24) }}
+                <span style="float:right;">
+                  <small><em class="text-muted"> {{comment.created | date}} </em></small>
+                </span>
+              </p>
+              <p> {{ comment.text_html|safe }} </p>
+            </div>
+            {% if current_user.is_authenticated and current_user == comment.user %}
+              <a href="{{ url_for('comment.delete', id=comment.id) }}"
+                 class="btn btn-danger btn-sm">
+                <span class="glyphicon glyphicon-remove"></span> {{ _('Delete comment') }}
```

Comment:
Insted of this, I'd prefer to just see the delete icon, like we see with votes.

Reply:
Added trash delete icon instead of explicit text.